tests/integration/test_timeseries_replicator.py

```diff
@@ -93,6 +91,8 @@ def test_timeseries_data_integration_service(
     azure_credential,
     remote_state_store,
 ):
+    # Run replicator before pushing data points in order to setup subscription
+    run_replicator(test_replicator)
```

Comment:
Do we need any assertions (or new tests) to assert proper creation of subscriptions and proper handling of edge conditions, e.g. multiple subscriptions existing?

Reply:
Added assert_subscription_created_in_cdf method